src/v2/Components/UserSettings/UserInformation/__tests__/UserInformation.jest.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { graphql } from "react-relay"
+import { createTestEnv } from "v2/DevTools/createTestEnv"
+import { UserInformationQueryResponse } from "v2/__generated__/UserInformationQuery.graphql"
+import { UserInformationRefetchContainer } from ".."
+import { UserInformationTestPage } from "./UserInformationTestPage"
+
+jest.unmock("react-relay")
+
+const setupTestEnv = () => {
+  return createTestEnv({
+    Component: (props: UserInformationQueryResponse) => (
+      <UserInformationRefetchContainer {...props} />
+    ),
+    TestPage: UserInformationTestPage,
+    defaultData: {
+      me: {
+        email: "foo@email.com",
+        internalID: "1234ABCD",
+        name: "Foo Bar",
+        paddleNumber: "12345",
+        phone: "555-123-4567",
+      },
+    },
+    defaultMutationResults: {
+      updateMyUserProfile: {},
+    },
+    query: graphql`
+      query UserInformationTestQuery {
+        me {
+          ...UserInformation_me
+        }
+      }
+    `,
+  })
+}
+
+describe("UserInformation", () => {
+  it("displays expected fields", async () => {
+    const env = setupTestEnv()
+    const page = await env.buildPage()
+
+    expect(page.submitButton.exists).toBeTruthy
+    expect(page.nameInput.exists).toBeTruthy
```

Comment:
#minor rather than asserting the existence, maybe better to assert on the contents (matching the fabricated data in the test setup).